### DISCUSSION TITLE
SO version upgrade from 2.3.52 with soup

### DISCUSSION THREAD

**Discussion by sikelos** [Security-Onion-Solutions/securityonion#8257] [Unsupported Versions] [answered]
Hi all,
I am trying to update our current SO version 2.3.52 -Standalone - Airgap installation that runs in CENTOS 7 box. I used soup that finished ok (unless I missed something) but still the /etc/soversion still shows vesrion 2.3.52.
What is the exact process for the SO version upgrade that works? 
Any suggestions would be much appreciated as I am running into circles on this.

**Comment by dougburks** [comment, marked as answer]
> still the /etc/soversion still shows vesrion 2.3.52.

If `/etc/soversion` still shows version 2.3.52, then EITHER the update didn't complete successfully OR you still had the 2.3.52 ISO image mounted and so there was nothing for soup to do.

You can run the `mount` command to see if you perhaps have an old ISO image mounted. If so, you can either `umount` the old ISO image or simply reboot the box.

You can also check the soup log in `/root/soup.log` for any additional clues.

> What is the exact process for the SO version upgrade that works?
https://docs.securityonion.net/en/2.3/soup.html
